azure_arc_servers_jumpstart/proxy/artifacts/install_arc_agent_proxy.sh

```diff
@@ -0,0 +1,101 @@
+# Get Environment Variables
+
+USER=$1
+SUBSCRIPTIONID=$2
+APPID=$3
+SP_PASSWORD=$4
+TENANTID=$5
+RG=$6
+LOCATION=$7
+VMNAME=$8
+URL=$9
+PORT=3128
+PASSWORD=$10
+
+touch /home/$USER/.bash_profile
+chmod +x /home/$USER/.bash_profile
+
+cat <<EOT > /home/$USER/.bash_profile
+#!/bin/bash
+##Environment Variables
+export SUBSCRIPTIONID=$SUBSCRIPTIONID
+export APPID=$APPID
+export SP_PASSWORD=$SP_PASSWORD
+export TENANTID=$TENANTID
+export RG=$RG
+export LOCATION=$LOCATION
+export VMNAME=$VMNAME
+export URL=$URL
+export PORT=$PORT
+export PASSWORD=$PASSWORD
+
+export HTTP_PROXY="http://$URL:$PORT"
+export https_proxy="http://$URL:$PORT"
```

Comment:
why dup?

Reply:
I need to have both one for HTTP and HTTPS connections they are both used by the OS. I'll change the capitalization